FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testViewSnapshot
+{
+    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    redView.backgroundColor = [UIColor redColor];
+    FBSnapshotVerifyView(redView, nil);
```

Comment:
Good question, will have to investigate..
On Sun, Dec 13, 2015 at 17:30 Patrick Chamelo <notifications@github.com>
wrote:

> In
> FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m
> <https://github.com/facebook/ios-snapshot-test-case/pull/137#discussion_r47451523>
> :
>
> > +@end
> > +
> > +@implementation FBSnapshotTestCaseDeviceAgnosticTests
> > +
> > +- (void)setUp
> > +{
> > +    [super setUp];
> > +    self.deviceAgnostic = YES;
> > +    self.recordMode = NO;
> > +}
> > +
> > +- (void)testViewSnapshot
> > +{
> > +    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
> > +    redView.backgroundColor = [UIColor redColor];
> > +    FBSnapshotVerifyView(redView, nil);
>
> Travis is hitting the assertion that you added, why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ios-snapshot-test-case/pull/137/files#r47451523>
> .
>